owncloudApp/src/main/java/com/owncloud/android/presentation/ui/authentication/LoginActivity.kt

```diff
@@ -417,7 +423,8 @@ class LoginActivity : AppCompatActivity(), SslUntrustedCertDialog.OnSslUntrusted
     private fun performGetAuthorizationCodeRequest(authorizationServiceConfiguration: AuthorizationServiceConfiguration) {
         Timber.d("A browser should be opened now to authenticate this user.")
         val clientId = getString(R.string.oauth2_client_id)
-        val redirectUri = Uri.parse(getString(R.string.oauth2_redirect_uri))
+        val redirectString = getString(R.string.oauth2_redirect_uri_scheme) + "://" + getString(R.string.oauth2_redirect_uri_path)
```

Comment:
Do you think we need a constant for `://` ?

Reply:
I see, we will build the redirect uri using uri builder.